app/javascript/packs/application.js

```diff
@@ -9,6 +9,11 @@ import "channels"
 
 import "bootstrap"
 
+document.addEventListener("turbolinks:load", () => {
```

Comment:
Uuuu, first javascript, nice ;)

Reply:
Karoč turbolinks ir īpašs rails radījums, kurš dzima no tādas pjax fiškas. Bet lai to saprastu, tev droši vien vēl vispirms jāpalas kas ir AJAX un XMLHttpRequest un ko tie dod.

īsumā sen sen agrāk lapas ielādējās un viss pasākums ar to arī beidzās, nākamie pieprasījumi uz serveri bija vnk lapas new load, bet ap kkādu 200x gadu uzplaukumu piedzīvoja AJAX - iespēja no Javascript koda izpildīt pieprasījjumu uz serveri un tad nu dzima super dinamiskās lapas  - kur varēja ielādēt jaunu saturu klāt bez actual browser refresh vai tml.

Reply:
Godigi sakot, man nav ne mazakas nojausmas kas saja koda dala notiek. Es iekopeju ar ceribu ka saks stradat Dropdowns, beigas aizmirsu iznemt koda dalu.